### DISCUSSION TITLE
[A-L] (2023/24) Foglio 8 - Esercizio 1

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#184] [Esercizi - Bertini]
![Screenshot 2023-11-20 223922](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/7fd12595-1881-413b-a3e5-b389ef437991)

**Comment by Elia-Belli** [comment]
Sia $X\sim Poisson(np)$ la v.a. che indica il numero di persone mancine tra $n=100$ e $p=0.02$

$$\mathbb{P}(X\geq 3)=1-e^{-2}\sum^2_{k=0} \frac{2^k}{k!}=1-\frac{1+2+2^2/2}{e^2}=1-\frac{5}{e^2}$$

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/79638284/31da2156-caf4-46f8-b06b-512f093faabc)
